### PR TITLE
Add entity copy success toaster

### PR DESCRIPTION
**Pull request by DaryaLari**

### CODE REVIEW

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10129-1810/component-entry-context-menu-view-1810/ru/">component.entry-context-menu.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10129-1810/component-entry-context-menu-view-1810/en/">component.entry-context-menu.view (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>Errors occurred when merging changes from your branch with the Weblate branch.</summary>

```
Rebasing (1/3)
Auto-merging src/i18n-keysets/component.entry-context-menu.view/en.json
CONFLICT (add/add): Merge conflict in src/i18n-keysets/component.entry-context-menu.view/en.json
Auto-merging src/i18n-keysets/component.entry-context-menu.view/ru.json
CONFLICT (add/add): Merge conflict in src/i18n-keysets/component.entry-context-menu.view/ru.json
Auto-merging src/server/components/charts-engine/components/chart-validator.ts
CONFLICT (add/add): Merge conflict in src/server/components/charts-engine/components/chart-validator.ts
Auto-merging src/shared/constants/entry.ts
CONFLICT (add/add): Merge conflict in src/shared/constants/entry.ts
Auto-merging src/shared/constants/qa/editor.ts
CONFLICT (add/add): Merge conflict in src/shared/constants/qa/editor.ts
Auto-merging src/shared/types/feature.ts
CONFLICT (add/add): Merge conflict in src/shared/types/feature.ts
Auto-merging src/shared/types/widget.ts
CONFLICT (add/add): Merge conflict in src/shared/types/widget.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/helpers/chartkit-adapter.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/helpers/chartkit-adapter.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/plugins/index.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/plugins/index.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/types.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/types.ts
Auto-merging src/ui/libs/DatalensChartkit/modules/html-generator/constants.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/modules/html-generator/constants.ts
Auto-merging src/ui/libs/DatalensChartkit/types/widget.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/types/widget.ts
Auto-merging src/ui/units/dash/modules/constants.ts
CONFLICT (add/add): Merge conflict in src/ui/units/dash/modules/constants.ts
Auto-merging src/ui/units/editor/configs/chart/chart-editor-types.ts
CONFLICT (add/add): Merge conflict in src/ui/units/editor/configs/chart/chart-editor-types.ts
error: could not apply f022023b4... Add entity copy success toaster
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply f022023b4... Add entity copy success toaster
 (1)
```

</details>

**Resolve conflicts according to instructions**
1. Switch to the current branch associated with this pull request.
```
git checkout CHARTS-10129
```
2. Add Weblate as remote. And remove previous upstream with the same name:
```
git remote remove weblate 2> /dev/null
git remote add weblate https://weblate.gravity-ui.com/git/datalens/charts-10129-1810/chartkit-1810/
git -c http.sslVerify=false remote update weblate
```
3. Merge Weblate changes:
```
git merge weblate/CHARTS-10129
```
4. Resolve conflicts:
```
edit ...
git add ...
git commit
```
5. Push changes to upstream repository, Weblate will fetch merge from there:
```
git push origin
```

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>Errors occurred when merging changes from your branch with the Weblate branch.</summary>

```
warning: skipped previously applied commit f06f9beb4
warning: skipped previously applied commit 60bf6250c
hint: use --reapply-cherry-picks to include skipped commits
hint: Disable this message with "git config advice.skippedCherryPicks false"
Rebasing (1/1)
Auto-merging .github/workflows/i18n-check.yml
CONFLICT (add/add): Merge conflict in .github/workflows/i18n-check.yml
Auto-merging src/i18n-keysets/component.entry-context-menu.view/en.json
CONFLICT (add/add): Merge conflict in src/i18n-keysets/component.entry-context-menu.view/en.json
Auto-merging src/i18n-keysets/component.entry-context-menu.view/ru.json
CONFLICT (add/add): Merge conflict in src/i18n-keysets/component.entry-context-menu.view/ru.json
Auto-merging src/server/components/charts-engine/components/chart-validator.ts
CONFLICT (add/add): Merge conflict in src/server/components/charts-engine/components/chart-validator.ts
Auto-merging src/shared/constants/entry.ts
CONFLICT (add/add): Merge conflict in src/shared/constants/entry.ts
Auto-merging src/shared/constants/qa/editor.ts
CONFLICT (add/add): Merge conflict in src/shared/constants/qa/editor.ts
Auto-merging src/shared/types/feature.ts
CONFLICT (add/add): Merge conflict in src/shared/types/feature.ts
Auto-merging src/shared/types/widget.ts
CONFLICT (add/add): Merge conflict in src/shared/types/widget.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/helpers/chartkit-adapter.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/helpers/chartkit-adapter.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/plugins/index.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/plugins/index.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/types.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/types.ts
Auto-merging src/ui/libs/DatalensChartkit/modules/html-generator/constants.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/modules/html-generator/constants.ts
Auto-merging src/ui/libs/DatalensChartkit/types/widget.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/types/widget.ts
Auto-merging src/ui/units/dash/modules/constants.ts
CONFLICT (add/add): Merge conflict in src/ui/units/dash/modules/constants.ts
Auto-merging src/ui/units/editor/configs/chart/chart-editor-types.ts
CONFLICT (add/add): Merge conflict in src/ui/units/editor/configs/chart/chart-editor-types.ts
error: could not apply f022023b4... Add entity copy success toaster
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply f022023b4... Add entity copy success toaster
 (1)
```

</details>

**Resolve conflicts according to instructions**
1. Switch to the current branch associated with this pull request.
```
git checkout CHARTS-10129
```
2. Add Weblate as remote. And remove previous upstream with the same name:
```
git remote remove weblate 2> /dev/null
git remote add weblate https://weblate.gravity-ui.com/git/datalens/charts-10129-1810/chartkit-1810/
git -c http.sslVerify=false remote update weblate
```
3. Merge Weblate changes:
```
git merge weblate/CHARTS-10129
```
4. Resolve conflicts:
```
edit ...
git add ...
git commit
```
5. Push changes to upstream repository, Weblate will fetch merge from there:
```
git push origin
```

File: src/ui/units/dash/containers/Body/Body.tsx [hunk 1]
```diff
@@ -858,13 +874,14 @@ class Body extends React.PureComponent<BodyProps> {
             dashkitSettings,
         } = this.props;
 
+        const {getPreparedCopyItemOptions: getOptions, ...context} = this.getContext();
```
Comment (flops) [inline src/ui/units/dash/containers/Body/Body.tsx:877]: Let's remove this method from memo https://github.com/datalens-tech/datalens-ui/blob/bf7e5a6be4a303058810be7264b47a262152ea98/src/ui/units/dash/containers/Body/Body.tsx#L705-L706
And use this `fn` https://github.com/datalens-tech/datalens-ui/blob/bf7e5a6be4a303058810be7264b47a262152ea98/src/ui/units/dash/containers/Body/Body.tsx#L694-L695 as class method that would be passed as `getPreparedCopyItemOptions` prop

**Review by flops — CHANGES_REQUESTED** [review]

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>Errors occurred when merging changes from your branch with the Weblate branch.</summary>

```
warning: skipped previously applied commit 70c3cb549
warning: skipped previously applied commit 06e65dce8
warning: skipped previously applied commit d2909a47c
warning: skipped previously applied commit f06f9beb4
warning: skipped previously applied commit 60bf6250c
hint: use --reapply-cherry-picks to include skipped commits
hint: Disable this message with "git config advice.skippedCherryPicks false"
Rebasing (1/1)
Auto-merging .github/workflows/i18n-check.yml
CONFLICT (add/add): Merge conflict in .github/workflows/i18n-check.yml
Auto-merging src/i18n-keysets/component.entry-context-menu.view/en.json
CONFLICT (add/add): Merge conflict in src/i18n-keysets/component.entry-context-menu.view/en.json
Auto-merging src/i18n-keysets/component.entry-context-menu.view/ru.json
CONFLICT (add/add): Merge conflict in src/i18n-keysets/component.entry-context-menu.view/ru.json
Auto-merging src/server/components/charts-engine/components/chart-validator.ts
CONFLICT (add/add): Merge conflict in src/server/components/charts-engine/components/chart-validator.ts
Auto-merging src/server/components/charts-engine/components/storage/types.ts
CONFLICT (add/add): Merge conflict in src/server/components/charts-engine/components/storage/types.ts
Auto-merging src/server/components/charts-engine/controllers/embeds.ts
CONFLICT (add/add): Merge conflict in src/server/components/charts-engine/controllers/embeds.ts
Auto-merging src/server/configs/opensource/common.ts
CONFLICT (add/add): Merge conflict in src/server/configs/opensource/common.ts
Auto-merging src/shared/constants/entry.ts
CONFLICT (add/add): Merge conflict in src/shared/constants/entry.ts
Auto-merging src/shared/constants/qa/editor.ts
CONFLICT (add/add): Merge conflict in src/shared/constants/qa/editor.ts
Auto-merging src/shared/schema/us/types/embeds.ts
CONFLICT (add/add): Merge conflict in src/shared/schema/us/types/embeds.ts
Auto-merging src/shared/types/charts.ts
CONFLICT (add/add): Merge conflict in src/shared/types/charts.ts
Auto-merging src/shared/types/common.ts
CONFLICT (add/add): Merge conflict in src/shared/types/common.ts
Auto-merging src/shared/types/feature.ts
CONFLICT (add/add): Merge conflict in src/shared/types/feature.ts
Auto-merging src/shared/types/widget.ts
CONFLICT (add/add): Merge conflict in src/shared/types/widget.ts
Auto-merging src/ui/components/Widgets/Chart/ChartWidget.tsx
CONFLICT (add/add): Merge conflict in src/ui/components/Widgets/Chart/ChartWidget.tsx
Auto-merging src/ui/components/Widgets/Chart/components/Header.tsx
CONFLICT (add/add): Merge conflict in src/ui/components/Widgets/Chart/components/Header.tsx
Auto-merging src/ui/constants/common.ts
CONFLICT (add/add): Merge conflict in src/ui/constants/common.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/helpers/chartkit-adapter.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/helpers/chartkit-adapter.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/plugins/index.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/plugins/index.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/types.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/types.ts
Auto-merging src/ui/libs/DatalensChartkit/components/ChartKitBase/components/Header/components/Menu/Items/DownloadCsv/DownloadCsv.tsx
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/components/ChartKitBase/components/Header/components/Menu/Items/DownloadCsv/DownloadCsv.tsx
Auto-merging src/ui/libs/DatalensChartkit/components/ChartKitBase/components/Header/components/Menu/Items/Export/Export.tsx
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/components/ChartKitBase/components/Header/components/Menu/Items/Export/Export.tsx
Auto-merging src/ui/libs/DatalensChartkit/components/ChartKitBase/components/Header/components/Menu/Items/Export/types.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/components/ChartKitBase/components/Header/components/Menu/Items/Export/types.ts
Auto-merging src/ui/libs/DatalensChartkit/modules/data-provider/charts/index.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/modules/data-provider/charts/index.ts
Auto-merging src/ui/libs/DatalensChartkit/modules/data-provider/charts/node.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/modules/data-provider/charts/node.ts
Auto-merging src/ui/libs/DatalensChartkit/modules/data-provider/charts/types.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/modules/data-provider/charts/types.ts
Auto-merging src/ui/libs/DatalensChartkit/modules/data-provider/charts/wizard.js
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/modules/data-provider/charts/wizard.js
Auto-merging src/ui/libs/DatalensChartkit/modules/export/export.js
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/modules/export/export.js
Auto-merging src/ui/libs/DatalensChartkit/modules/html-generator/constants.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/modules/html-generator/constants.ts
Auto-merging src/ui/libs/DatalensChartkit/types/widget.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/types/widget.ts
Auto-merging src/ui/registry/units/chart/types/functions/getChartkitMenuByType.ts
CONFLICT (add/add): Merge conflict in src/ui/registry/units/chart/types/functions/getChartkitMenuByType.ts
Auto-merging src/ui/units/dash/containers/Body/Body.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Body/Body.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Control2.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Control2.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/AppearanceSection/Rows/HintRow/HintRow.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/AppearanceSection/Rows/HintRow/HintRow.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/AppearanceSection/Rows/InnerTitleRow/InnerTitleRow.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/AppearanceSection/Rows/InnerTitleRow/InnerTitleRow.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/AppearanceSection/Rows/TitlePlacementRow/TitlePlacementRow.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/AppearanceSection/Rows/TitlePlacementRow/TitlePlacementRow.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/AppearanceSection/Rows/TitleRow/TitleRow.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/AppearanceSection/Rows/TitleRow/TitleRow.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/CommonSettingsSection.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/CommonSettingsSection.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ConnectionSettings/ConnectionSettings.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ConnectionSettings/ConnectionSettings.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ConnectionSettings/components/ConnectionSelector/ConnectionSelector.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ConnectionSettings/components/ConnectionSelector/ConnectionSelector.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ConnectionSettings/components/QueryTypeControl/QueryTypeControl.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ConnectionSettings/components/QueryTypeControl/QueryTypeControl.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ConnectionSettings/components/QueryTypeControl/components/EditLabelControl/EditLabelControl.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ConnectionSettings/components/QueryTypeControl/components/EditLabelControl/EditLabelControl.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ConnectionSettings/components/QueryTypeControl/components/EditQueryControl/EditQueryControl.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ConnectionSettings/components/QueryTypeControl/components/EditQueryControl/EditQueryControl.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/DatasetSelector/DatasetSelector.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/DatasetSelector/DatasetSelector.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ExternalSelectorSettings/ExternalSelectorSettings.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ExternalSelectorSettings/ExternalSelectorSettings.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/InputTypeSelector/InputTypeSelector.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/InputTypeSelector/InputTypeSelector.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ParameterNameInput/ParameterNameInput.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/CommonSettingsSection/ParameterNameInput/ParameterNameInput.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/OperationSelector/OperationSelector.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/OperationSelector/OperationSelector.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/ParametersSection/ParametersSection.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/ParametersSection/ParametersSection.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/Acceptable/Acceptable.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/Acceptable/Acceptable.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/ListValueControl.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/ListValueControl.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/MultiselectableCheckbox.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/MultiselectableCheckbox.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/ValueSelector/DynamicValueSelect.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/ValueSelector/DynamicValueSelect.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/ValueSelector/StaticValueSelect.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/ValueSelector/StaticValueSelect.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/ValueSelector/hooks.ts
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ListValueControl/ValueSelector/hooks.ts
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/RequiredValueCheckbox/RequiredValueCheckbox.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/RequiredValueCheckbox/RequiredValueCheckbox.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ValueSelector.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/Sections/ValueSelector/ValueSelector.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Control2/SelectorTypeSelect/SelectorTypeSelect.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Control2/SelectorTypeSelect/SelectorTypeSelect.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/DialogEditQuery/DialogEditQuery.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/DialogEditQuery/DialogEditQuery.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/GroupControl/ControlsPlacementDialog/ControlsPlacementDialog.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/GroupControl/ControlsPlacementDialog/ControlsPlacementDialog.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/GroupControl/GroupControl.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/GroupControl/GroupControl.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/GroupControl/GroupControlBody/GroupControlBody.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/GroupControl/GroupControlBody/GroupControlBody.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/GroupControl/GroupControlSidebar/GroupControlSidebar.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/GroupControl/GroupControlSidebar/GroupControlSidebar.tsx
Auto-merging src/ui/units/dash/modules/constants.ts
CONFLICT (add/add): Merge conflict in src/ui/units/dash/modules/constants.ts
Auto-merging src/ui/units/dash/store/actions/controls/actions.ts
CONFLICT (add/add): Merge conflict in src/ui/units/dash/store/actions/controls/actions.ts
Auto-merging src/ui/units/dash/store/actions/dashTyped.ts
CONFLICT (add/add): Merge conflict in src/ui/units/dash/store/actions/dashTyped.ts
Auto-merging src/ui/units/dash/store/actions/index.ts
CONFLICT (add/add): Merge conflict in src/ui/units/dash/store/actions/index.ts
Auto-merging src/ui/units/dash/store/reducers/dash.js
CONFLICT (add/add): Merge conflict in src/ui/units/dash/store/reducers/dash.js
Auto-merging src/ui/units/dash/store/reducers/dashTypedReducer.ts
CONFLICT (add/add): Merge conflict in src/ui/units/dash/store/reducers/dashTypedReducer.ts
Auto-merging src/ui/units/dash/store/selectors/dashTypedSelectors.ts
CONFLICT (add/add): Merge conflict in src/ui/units/dash/store/selectors/dashTypedSelectors.ts
Auto-merging src/ui/units/editor/configs/chart/chart-editor-types.ts
CONFLICT (add/add): Merge conflict in src/ui/units/editor/configs/chart/chart-editor-types.ts
error: could not apply f022023b4... Add entity copy success toaster
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply f022023b4... Add entity copy success toaster
 (1)
```

</details>

**Resolve conflicts according to instructions**
1. Switch to the current branch associated with this pull request.
```
git checkout CHARTS-10129
```
2. Add Weblate as remote. And remove previous upstream with the same name:
```
git remote remove weblate 2> /dev/null
git remote add weblate https://weblate.gravity-ui.com/git/datalens/charts-10129-1810/chartkit-1810/
git -c http.sslVerify=false remote update weblate
```
3. Merge Weblate changes:
```
git merge weblate/CHARTS-10129
```
4. Resolve conflicts:
```
edit ...
git add ...
git commit
```
5. Push changes to upstream repository, Weblate will fetch merge from there:
```
git push origin
```

**Review by flops — APPROVED** [review]

**Pull request merged**